README.md

```diff
@@ -184,7 +184,7 @@ If you have direct access to the underlying Kubernetes Nodes in your cluster, CO
 2. Run the upgrade script to patch the CORTX on Kubernetes Deployments using an updated image _(:information_source: You will want to update the `TARGET_IMAGE` variable below to your desired image tag)_. The script will stop all CORTX Pods, update the Deployments, and then re-start the Pods.
 
    ```bash
-   TARGET_IMAGE="ghcr.io/seagate/cortx-all:2.0.0-641-custom-ci"
+   TARGET_IMAGE="ghcr.io/seagate/cortx-data:2.0.0-790"
```

Comment:
It seems simpler to me to list "cortx-data" as the target image rather than explain that it can be cortx-<anything>.